staging/src/k8s.io/client-go/kubernetes/fake/clientset_generated.go

```diff
@@ -57,7 +57,7 @@ import (
 // without applying any validations and/or defaults. It shouldn't be considered a replacement
 // for a real clientset and is mostly useful in simple unit tests.
 func NewSimpleClientset(objects ...runtime.Object) *Clientset {
-	o := testing.NewObjectTracker(api.Scheme, api.Codecs.UniversalDecoder())
+	o := testing.NewObjectTracker(api.Registry, api.Scheme, api.Codecs.UniversalDecoder())
```

Comment:
This fixes one of the tests.  This will get updated on the next re-sync, but I don't want to wait on it here.